examples/java_postgresql_customvar/README.md

```diff
@@ -194,19 +249,25 @@ spec:
     group: apps
     version: v1
     resource: deployments
-  backingServiceSelector:
-    group: postgresql.baiju.dev
-    version: v1alpha1
-    kind: Database
-    resourceRef: db-demo
-    id: postgresDB
+  backingServiceSelectors:
+  	- group: postgresql.baiju.dev
```

Comment:
indendation of gvk should all lie in the same line

Reply:
@Avni-Sharma Fixed it here 449b58e